src/test/java/lotto/domain/RanksTest.java

```diff
@@ -0,0 +1,39 @@
+package lotto.domain;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("순위 collection 테스트")
+class RanksTest {
+
+	@Test
+	@DisplayName("순위 collection 생성")
+	void createRanks() {
+		Ranks ranks = Ranks.of(List.of(6, 5, 3));
+		assertThat(ranks).isInstanceOf(Ranks.class);
+	}
+
+	@Test
+	@DisplayName("FOURTH 부터 역순으로 정렬 된 순위 별 개수 map 반환")
+	void groupByTest() {
+		Ranks ranks = Ranks.of(List.of(6, 5, 4, 6, 6));
+		Map<Rank, Long> groupBy = ranks.groupBy();
+		assertAll(
+			() -> assertThat(groupBy).containsEntry(Rank.FOURTH, 0L),
+			() -> assertThat(groupBy).containsEntry(Rank.THIRD, 1L),
+			() -> assertThat(groupBy).containsEntry(Rank.SECOND, 1L),
+			() -> assertThat(groupBy).containsEntry(Rank.FIRST, 3L),
+			() -> assertThat(groupBy.keySet().toArray()[0]).isEqualTo(Rank.FOURTH),
+			() -> assertThat(groupBy.keySet().toArray()[1]).isEqualTo(Rank.THIRD),
+			() -> assertThat(groupBy.keySet().toArray()[2]).isEqualTo(Rank.SECOND),
+			() -> assertThat(groupBy.keySet().toArray()[3]).isEqualTo(Rank.FIRST)
+		);
+	}
```

Comment:
`containsExactlyEntriesOf` 을 이용하시면 내부에서 순서도 같이 검증해주기 때문에
반복되는 검증 메서드 호출 없이 원하시는 검증을 그대로 수행할 수 있을 것 같아요!!
이렇게 검증할 수도 있으니 참고만 해주세요!! 😁
```suggestion
	void groupByTest() {
		//given
		Ranks ranks = Ranks.of(List.of(6, 5, 4, 6, 6));
		LinkedHashMap<Rank, Long> expected = new LinkedHashMap<>();
		expected.put(Rank.FOURTH, 0L);
		expected.put(Rank.THIRD, 1L);
		expected.put(Rank.SECOND, 1L);
		expected.put(Rank.FIRST, 3L);
		//when
		Map<Rank, Long> groupBy = ranks.groupBy();
		//then
		assertThat(groupBy).containsExactlyEntriesOf(expected);
	}
```

Reply:
저런 메서드가 있는 지 몰랐네요 😂 
그리고 테스트 메서드가 조금 길어졌을 때, 
`given, when, then` 으로 나누니 훨씬 가독성이 좋아지네요!!
좋은 조언 감사드립니다!